.github/workflows/run-cypress-tests.yml

```diff
@@ -0,0 +1,13 @@
+name: Cypress tests
+on: [pull_request]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Is there a reason we're not using [latest (v3)](https://github.com/actions/checkout#usage) here? Looks like the [main diff](https://github.com/actions/checkout/blob/main/CHANGELOG.md) is updating to node 16 and various dep updates - if cypress runs on it would be nice to start at latest.